nrf_security/doc/driver_config.rst

```diff
@@ -0,0 +1,344 @@
+   * The :ref:`nrf_security_drivers_cc3xx` does not provide hardware support for AES GCM on devices with Arm CryptoCell cc310.
+
+
+ECC configurations
+******************
+
+Elliptic Curve Cryptography (ECC) can be enabled by setting one or more Kconfig variables in the following table:
+
++-----------------------+---------------------------------------------+
+| ECC algorithm         | Configurations                              |
++=======================+=============================================+
+| ECDH                  | :kconfig:`PSA_WANT_ALG_ECDH`                |
++----------------+----------------------------------------------------+
+| ECDSA                 | :kconfig:`PSA_WANT_ALG_ECDSA`               |
++-----------------------+---------------------------------------------+
+| ECDSA (deterministic) | :kconfig:`PSA_WANT_ALG_DETERMINISTIC_ECDSA` |
++-----------------------+---------------------------------------------+
+
+The ECC algorithm support is dependent on one or more Kconfig variables enabling curve support according to :ref:`ECC curve configurations`.
+
+
+ECC driver configurations
+=========================
+
+The Kconfig variables in the following table can be used for fine-grained control over which drivers provides ECC support:
+
++-----------------------+------------------------------------------------------------+-------------------------------------------------------------+
+| ECC algorithm         | nrf_cc3xx driver support                                   | nrf_oberon driver support                                   |
++=======================+============================================================+=============================================================+
+| ECDH                  | :kconfig:`PSA_CRYPTO_DRIVER_ALG_ECDH_CC3XX`                | :kconfig:`PSA_CRYPTO_DRIVER_ALG_ECDSA`                      |
++----------------+-------------------------------------------------------------------+-------------------------------------------------------------+
+| ECDSA                 | :kconfig:`PSA_CRYPTO_DRIVER_ALG_ECDSA_CC3XX`               | :kconfig:`PSA_CRYPTO_DRIVER_ALG_ECDSA_OBERON`               |
++-----------------------+------------------------------------------------------------+-------------------------------------------------------------+
+| ECDSA (deterministic) | :kconfig:`PSA_CRYPTO_DRIVER_ALG_DETERMINISTIC_ECDSA_CC3XX` | :kconfig:`PSA_CRYPTO_DRIVER_ALG_DETERMINISTIC_ECDSA_OBERON` |
++-----------------------+------------------------------------------------------------+-------------------------------------------------------------+
+
+.. note::
+   * If an ECC algorithm is enabled and no PSA driver enables or supports it, then builtin Mbed TLS support is enabled and used.
+   * The :ref:`nrf_security_drivers_oberon` is currently limited to curve types secp224r1 and secp256r1 for ECDH and ECDSA.
```

Comment:
What about Curve25519 ?

Reply:
Curve25519 is not possible for the algorithms ECDH or ECDSA. The corresponding equivalents for Curve25519 are x25519 (equivalent to ECDH) and Ed25519 (Also known as EdDSA, equivalent to ECDSA). There is no standardized APIs for this, yet. Only base curve operations are allowed for Curve25519